docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -44,10 +44,10 @@ Change directories to *FSNetCore* and start adding projects to the solution fold
 Use the `dotnet new` command, create a Class Library project in the **src** folder named Library. 
 
 ```bash
-dotnet new classlib -lang F# -o src/Library 
+dotnet new lib -lang F# -o src/Library 
```

Comment:
I think I prefer the guidance being for "lib" as well.  [The template doesn't have a class in it either.](https://github.com/dotnet/templating/blob/rel/vs2017/3-Preview2/template_feed/Microsoft.DotNet.Common.ProjectTemplates.2.0/content/ClassLibrary-FSharp/Library.fs)